docs/RELEASE-NOTES.rst

```diff
@@ -2,22 +2,34 @@ Release Notes for Container Ingress Services for Kubernetes & OpenShift
 =======================================================================
 
 
-Next Release
+2.4.1
 -------------
 Added Functionality
 ```````````````````
+* CIS supports `F5 IPAM Controller 0.1.3 <https://github.com/F5Networks/f5-ipam-controller/blob/main/docs/RELEASE-NOTES.rst>`_.
+* Helm Chart Enhancements:
+    - Added support for multiple namespace configuration parameter with CIS operator.
 
 Bug Fixes
 `````````
-* :issues: `1737` Inconsistent ordering of policy rules when adding an Ingress path
+* :issues: `1737` Inconsistent ordering of policy rules when adding an Ingress path.
 * :issues: `1808` K8S BIG-IP Controller upload old certificate to BIG-IP.
-* Fix Deleting Stale Specs from IPAM CR when CIS restarts
-* Fix Populate allocated ip for virtual-server in the VSAddress colomn while using IPAM.
-* Fix Processing endpoint with node names when CIS in Cluster mode
-
-Limitations
-```````````
+* Stale IPAM CR configuration is deleted when CIS restarts.
+* IPAM allocated IP address now populates for VirtualServer under VSAddress column.
+* CIS supports endpoints created with no nodeNames in Cluster mode for Headless Service.
```

Comment:
CIS supports endpoints created without nodeNames in Cluster mode for Headless Service.

Reply:
Done